### PR TITLE
Deprecate --format for podman top

### PR DESCRIPTION
**Pull request by baude**
Issue #169 describes a common failure when running podman top where
if the commands inside the container container a space in them, podman
will panic.  This was occuring because we take the output from ps and
attempt to format it nicely for output and things like JSON.  Given that
this cannot be predicted or dealt with programatically, the decision was
made to deprecate the format switch and simply output what ps provides
us.

Migrated top integration tests to ginkgo.

Resolves Issue: https://github.com/projectatomic/libpod/issues/169

Signed-off-by: baude <bbaude@redhat.com>

### CODE REVIEW

**Comment by mheon** [comment]
LGTM

**Comment by mheon** [comment]
@baude There are no backend changes here - was it enough to change podman only, or are there missing libpod changes?

**Comment by baude** [comment]
because this was --format, it's all in the frontend (i.e. podman)

**Comment by mheon** [comment]
Awesome. LGTM.

**Comment by rhatdan** [comment]
@rh-atomic-bot r+

File: test/e2e/top_test.go [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+		Expect(result.ExitCode()).To(Equal(125))
+	})
+
+	It("podman top on bogus container", func() {
+		result := podmanTest.Podman([]string{"top", "1234"})
+		result.WaitWithDefaultTimeout()
+		Expect(result.ExitCode()).To(Equal(125))
+	})
+
+	It("podman top on non-running container", func() {
```
Comment (TomSweeneyRedHat) [inline test/e2e/top_test.go]: The names are more generic than they used to be.  Can we append 'by id with defaults" here?

File: test/e2e/top_test.go [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+		_, ec, cid := podmanTest.RunLsContainer("")
+		Expect(ec).To(Equal(0))
+		result := podmanTest.Podman([]string{"top", cid})
+		result.WaitWithDefaultTimeout()
+		Expect(result.ExitCode()).To(Equal(125))
+	})
+
+	It("podman top on container", func() {
```
Comment (TomSweeneyRedHat) [inline test/e2e/top_test.go]: container to "running container".

File: test/e2e/top_test.go [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+		session.WaitWithDefaultTimeout()
+		Expect(session.ExitCode()).To(Equal(0))
+
+		result := podmanTest.Podman([]string{"top", "-l"})
+		result.WaitWithDefaultTimeout()
+		Expect(result.ExitCode()).To(Equal(0))
+		Expect(len(result.OutputToStringArray())).To(BeNumerically(">", 1))
+	})
+
+	It("podman top on non-running container", func() {
```
Comment (TomSweeneyRedHat) [inline test/e2e/top_test.go]: This looks to be a running container.  Did you mean to do a create here?

**Pull request closed without merging**